scripts/in_container/prod/clean-logs.sh

```diff
@@ -33,5 +32,5 @@ while true; do
   find "${DIRECTORY}"/logs -mtime +"${RETENTION}" -name '*.log' -delete
 
   seconds=$(( $(date -u +%s) % EVERY))
-  [[ $seconds -lt 1 ]] || sleep $((EVERY - seconds))
+  (( "${seconds}" < 1 )) || sleep $((EVERY - seconds))
```

Comment:
Why is the suggested syntax considered better than the current one?

Reply:
Actually I like it better. Explicit math bash expression is more readable than -lt

Reply:
However it should be simpler:

```suggestion
  (( seconds < 1 )) || sleep $((EVERY - seconds))
```

in math expression there is no need for either quoting or '$'

Reply:
Suggested by google cheat-sheet for numerical comparison and for better and simpler understanding